base/deprecated.jl

```diff
@@ -1341,7 +1341,7 @@ end
 @deprecate xor(A::AbstractArray, B::AbstractArray)  xor.(A, B)
 
 # QuadGK moved to a package (#19741)
-function quadgk(args...)
+function quadgk(args...; kwargs...)
     error(string(quadgk, args, " has been moved to the package QuadGK.jl.\n",
```

Comment:
I'm a littlr surprised this somehow puts parens around the argument like you'd want it to